src/faq/howto.rst

```diff
@@ -0,0 +1,22 @@
+.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _faq/howto:
+
+=======
+How-Tos
+=======
+
+How can I get a list of the design documents in a database?
+-----------------------------------------------------------
+
+Query the `_all_docs` view with `startkey="_design/"&endkey="_design0"`.
```

Comment:
We have a new `/_design_docs` endpoint, recommend that instead. It should be documented - and the FAQ should link to the documentation for the endpoint. If it's not documented, file a bug.